pkg/variantregistry/ocp_test.go

```diff
@@ -658,23 +659,22 @@ func TestVariantSyncer(t *testing.T) {
 		{
 			job: "periodic-ci-openshift-with-no-release-info",
 			expected: map[string]string{
-				VariantArch:             "amd64",
-				VariantInstaller:        "ipi",
-				VariantNetworkStack:     "ipv4",
-				VariantOwner:            "eng",
-				VariantSuite:            "unknown",
-				VariantTopology:         "ha",
-				VariantUpgrade:          VariantNoValue,
-				VariantAggregation:      VariantNoValue,
-				VariantProcedure:        "none",
-				VariantJobTier:          "standard",
-				VariantFeatureSet:       VariantDefaultValue,
-				VariantNetworkAccess:    VariantDefaultValue,
-				VariantScheduler:        VariantDefaultValue,
-				VariantSecurityMode:     VariantDefaultValue,
-				VariantContainerRuntime: "",
-				VariantCGroupMode:       "v2",
-				VariantLayeredProduct:   VariantNoValue,
+				VariantArch:           "amd64",
+				VariantInstaller:      "ipi",
+				VariantNetworkStack:   "ipv4",
+				VariantOwner:          "eng",
+				VariantSuite:          "unknown",
+				VariantTopology:       "ha",
+				VariantUpgrade:        VariantNoValue,
+				VariantAggregation:    VariantNoValue,
+				VariantProcedure:      "none",
+				VariantJobTier:        "standard",
+				VariantFeatureSet:     VariantDefaultValue,
+				VariantNetworkAccess:  VariantDefaultValue,
+				VariantScheduler:      VariantDefaultValue,
+				VariantSecurityMode:   VariantDefaultValue,
+				VariantCGroupMode:     "v2",
+				VariantLayeredProduct: VariantNoValue,
```

Comment:
It looks like VariantContainerRuntime disappeared here, which was empty value before. I'm vaguely remembering that we tried to make sure all jobs had all variants, not 100% sure why but I believe it was due to query concerns and behavior on the joins we do to get variants. Have you seen evidence we didn't stick to the above, or confirmed that this is safe to have a job that has NO value for a variant?

Reply:
Do we set other variants to ""? Normally they had a value which is why I removed it.  It doesn't seem to cause a problem.  There's other variants we don't set a value to (like VariantFromRelease).